lang/lowering/src/lower/exp/literal.rs

```diff
@@ -0,0 +1,46 @@
+use polarity_lang_miette_util::ToMiette;
+use polarity_lang_parser::cst::{self, Ident};
+
+use crate::{Ctx, DeclMeta, LoweringError, LoweringResult, lower::Lower};
+
+impl Lower for cst::exp::StrLit {
+    type Target = polarity_lang_ast::Exp;
+
+    fn lower(&self, ctx: &mut Ctx) -> LoweringResult<Self::Target> {
+        let cst::exp::StrLit { span, original, unescaped } = self;
```

Comment:
```suggestion
        let cst::exp::StringLit { span, original, unescaped } = self;
```
or `StringLiteral`? I would like to keep it consistent with the type name `String`.